src/App.java

```diff
@@ -1,5 +1,426 @@
+     * using-system-out-println
+     */
+
+    // Borders and symbols https://www.w3.org/TR/xml-entity-names/025.html, and
+    // https://www.alt-codes.net/arrow_alt_codes.php
+
+    private HashMap<String, Deck> decks;
+    private final Printer printer = new Printer();
+
+    /**
+     * Main method to start the application.
+     */
+    public void Start() {
+        //Get flashcards
+        decks = getDecks("flashcards.txt");
+        //Print Intro
+        printIntro();
+        printMenu();
+    }    
+
+    /**
+     * Print intro to program
+     */
+    public void printIntro() {
+        printer.output( BLUE + "╭───────────────────────────────────────────────────────────────────────────────────────────────╮\n" +
+                        "│  _______  _        _______  _______           _______  _______  _______  ______   _______  _  │\n" + //
+                        "│ (  ____ \\( \\      (  ___  )(  ____ \\|\\     /|(  ____ \\(  ___  )(  ____ )(  __  \\ (  ____ \\( ) │\n" + //
+                        "│ | (    \\/| (      | (   ) || (    \\/| )   ( || (    \\/| (   ) || (    )|| (  \\  )| (    \\/| | │\n" + //
+                        "│ | (__    | |      | (___) || (_____ | (___) || |      | (___) || (____)|| |   ) || (_____ | | │\n" + //
+                        "│ |  __)   | |      |  ___  |(_____  )|  ___  || |      |  ___  ||     __)| |   | |(_____  )| | │\n" + //
+                        "│ | (      | |      | (   ) |      ) || (   ) || |      | (   ) || (\\ (   | |   ) |      ) |(_) │\n" + //
+                        "│ | )      | (____/\\| )   ( |/\\____) || )   ( || (____/\\| )   ( || ) \\ \\__| (__/  )/\\____) | _  │\n" + //
+                        "│ |/       (_______/|/     \\|\\_______)|/     \\|(_______/|/     \\||/   \\__/(______/ \\_______)(_) │\n" + //
+                        "╰───────────────────────────────────────────────────────────────────────────────────────────────╯\n");
+                        printer.output("╭───────────────────────────────────────────────────────────────────────────────────────────────╮\n" +
+                        "│" + RESET + "                              Welcome to the Flashcard App!                                    " + BLUE + "│\n" + //
+                        "│" + RESET + "                      This app allows you to create and manage flashcards                      " + BLUE + "│\n" + //
+                        "│" + RESET + "                       You can add, remove, and view decks of flashcards.                      " + BLUE + "│\n" + //
+                        "│" + RESET + "                                   Let's get started!                                          " + BLUE + "│\n" + //
+                        "╰───────────────────────────────────────────────────────────────────────────────────────────────╯\n" + RESET);
+    }
+
+    /**
+     * Prints the main menu of the application.
+     */
+    public void printMenu() {

```

Comment:
I like your overall menu structure - using returns to exit back to the previous menu, and using a while (true) loop to repeat until a return is hit is pretty clever. And your use of switch statements is appropriate for the context. 

Some design structures would argue that having returns in the middle of the function is not great design, and you would be better off using a boolean flag variable and letting return happen by default. So instead of `while (true)` you could use `while (stillGoing)` and then instead of `return` use `stillGoing = false; break;` to exit the loop.